inputs/mvs_config.json

```diff
@@ -379,93 +335,97 @@
             "outflow_direction": "Electricity (DSO)",
             "peak_demand_pricing": {
                 "unit": "currency/kW",
-                "value": 0
+                "value": 60
             },
             "peak_demand_pricing_period": {
                 "unit": "times per year (1,2,3,4,6,12)",
-                "value": 1
+                "value": 2
```

Comment:
The old values would be better here, as they are more basic? On the other hand, if the MVS runs through with this json, then more functions are tested.

Reply:
What about keeping the old values for simplicity but adding a benchmark test with peak demand pricing? I also see the csvs and json as an example (Do you agree?) .. then it should be easy and as peak demand pricing is not very common -should not include it.

Reply:
Yeah, thats better probably.

Reply:
So it should be `1` instead of `2`?
btw the [description](https://mvs-eland.readthedocs.io/en/latest/MVS_parameters.html?highlight=maximumCap#energyproviders-csv) of peak_demand_pricing_period is not very understandable : "Only one of the following are acceptable values: 1 (yearly), 2, 3 ,4, 6, 12 (monthly)."

I understand the values not equal to 1 as "every two months", "every 3 months", but then "every 12 months" should be equivalent to 1. 
Or is it 2 --> twice per month, 3 --> three times per months? If yes, then it is called a frequency, not a period, because the unit is 1/time :)

Reply:
> So it should be `1` instead of `2`?

yes

> btw the [description](https://mvs-eland.readthedocs.io/en/latest/MVS_parameters.html?highlight=maximumCap#energyproviders-csv) of peak_demand_pricing_period is not very understandable : "Only one of the following are acceptable values: 1 (yearly), 2, 3 ,4, 6, 12 (monthly)."
> 
> I understand the values not equal to 1 as "every two months", "every 3 months", but then "every 12 months" should be equivalent to 1.
> Or is it 2 --> twice per month, 3 --> three times per months? If yes, then it is called a frequency, not a period, because the unit is 1/time :)

It should be 1/year, so if we use 1 it is 1 per year, 2 is twice per year.